zerver/lib/cache.py

```diff
@@ -205,7 +223,7 @@ def generic_bulk_cached_fetch(cache_key_function, query_function, object_ids,
     return dict((object_id, cached_objects[cache_keys[object_id]]) for object_id in object_ids
                 if cache_keys[object_id] in cached_objects)
 
-def cache(func):
+def cache(func): # type: ignore: https://www.python.org/dev/peps/pep-0484/#callable
```

Comment:
hmm why isn't the annotation `type: (Callable[Any, Any]) -> Callable[Any, Any]` ?